public/stylesheets/faq.css

```diff
@@ -25,8 +25,9 @@ figure figcaption {
 }
 
 .faq .faq-item h2 {
-    margin-top: 0;
+    margin-top: -70px;
```

Comment:
Yeah, through some research online I found that is the recommended fix. I will go ahead an fix #800 as well.